app/scripts/views/settings/emails.js

```diff
@@ -64,6 +69,14 @@ define(function (require, exports, module) {
       }
     },
 
+    _canChangePrimaryEmail () {
+      if (Url.searchParam('canChangeEmail', this.window.location.search)) {
```

Comment:
You can simplify this slightly be mixing in the [SearchParamMixin](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/search-param-mixin.js). This would collapse to:

```js
return !! this.getSearchParam('canChangeEmail');
```

Can you document this new param in [query-params.md under Experimental/development parameters](https://github.com/mozilla/fxa-content-server/blob/master/docs/query-params.md#experimentaldevelopment-parameters)?